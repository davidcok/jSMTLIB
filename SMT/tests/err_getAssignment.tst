; get-assignment without check-sat
(set-logic QF_UF)
(get-assignment)
(get-assignment xxx)
