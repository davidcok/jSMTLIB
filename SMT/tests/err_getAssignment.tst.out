success
(get-assignment)
^^^^^^^^^^^^^^^^
(error "The get-assignment command is only valid if :produce-assignments has been enabled")
(get-assignment xxx)
                ^^^
(error "A get-assignment command takes no arguments")
